### PR TITLE
Testing Page HTML and CSS Layout

### PR DESCRIPTION
**Pull request by chrisn02**
Have created the HTML and CSS layout for the testing page. The subject and test names are displayed in the top corner of the screen. There is space for the question to go with the question number as well as a progress bar underneath it. The answer boxes increase in size slightly when hovered over and while it has not been implemented yet I am planning on having it so that when you click on an answer a button will appear on the bottom right corner of the page to advance to the next question. As seen in the image there is some blank space but this was not there when I was originally making the page, so will have to adjust based on screen size.
TestingPage

### CODE REVIEW

**Review by gchodancar19 — APPROVED** [review]
Is there anything else to added like coding.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        padding-top: 10px;
+    }
+
+    .answers
+    {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        gap: 25px;
+        width: 75%;
+        height: 20%;
```
Comment (Jforjo) [inline style.css:239]: The answers box should also be taller, as I think each box is too small in height right now.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        text-align: center;
+        margin-bottom: 20px;
+        font-size: 20px;
+        padding-top: 10px;
+    }
+
+    .answers
+    {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
```
Comment (Jforjo) [inline style.css:236]: When the screen gets too small you should just use 1 column so the questions stack on top of each other.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+     {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        margin-bottom: 20px;
+        max-width: 900px;
+        min-width: 900px;
+        max-height: 100px;
+        min-height: 100px;
```
Comment (Jforjo) [inline style.css]: There is no need to have both `max-height` and `min-height` - just use `height`.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+    .question
+     {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        margin-bottom: 20px;
+        max-width: 900px;
+        min-width: 900px;
```
Comment (Jforjo) [inline style.css]: This `min-width` will cause the question to spill off the screen when the view-width gets too small. You can just remove this.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        margin: 0 auto;
+    }
+
+    .question
+     {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        margin-bottom: 20px;
```
Comment (Jforjo) [inline style.css]: This is redundant as you have `margin: 0 auto;` below, which overwrites this `margin-bottom`.

**Review by Jforjo — CHANGES_REQUESTED** [review]
See individual line comments.
Also, you could try to get the progress bar working by putting a `<i>` tag in with `inset: 0` and a percentage width. This percentage width being the amount the progress bar is filled.

**Comment by chrisn02** [comment]
Thanks for your feedback, I will work on this now.

**Comment by chrisn02** [comment]
Have made the requested changes. Have made the answer boxes taller and when on a smaller screen such as mobile, the boxes will stack on top of each other. The progress bar now fills based on percentage, the image is on 75% as an example. Let me you what you think.
TestingPage_Fix
TestingPage_MobileView

**Review by Jforjo — APPROVED** [review]
Looks good, but should should also make sure it looks good if there are only 2 or 3 answers too.

**Review by Jforjo — APPROVED** [review]
I don't think you can merge something if you've also made changes, since we have to approve it again. You've added a new line with nothing on it, which is a change that we need to review. When merging make sure there are no changes next time.

**Comment by chrisn02** [comment]
I think that must've happened as I was resolving a merge conflict, apologies

**Pull request merged**